caps/capabilities.go

```diff
@@ -42,3 +46,22 @@ type Capability struct {
 func (c Capability) String() string {
 	return c.Name
 }
+
+// SetAttr sets capability attribute to a given value.
+// TODO: remove temporary function implementation once attrtypes are merged.
```

Comment:
Can we please drop these temporary methods altogether for the time being?  They're purely a setter and a getter for a public map attribute, with literally zero benefit. That means we're simply adding cruft that needs to be removed later and that is hard to agree or disagree because the full replacement TODOs give them a blank check of existence.